src/code/ucode_disas.c

```diff
@@ -1185,7 +1008,7 @@ void UCodeDisas_Disassemble(UCodeDisas* this, GfxMod* ptr) {
 
                                     case G_MV_MATRIX: {
                                         DISAS_LOG("gsSPForceMatrix(0x%08x),", movemem.data);
-                                        ptr += 1;
+                                        ptr += 2 - 1;
```

Comment:
Moreso just out of curisoity why the `+= 2 - 1`, instead of just `++`?

Reply:
`gsSPForceMatrix` is two `Gfx`
https://github.com/zeldaret/oot/blob/b806ac0191764ff3a68076d9fb0543a1135ebde7/include/ultra64/gbi.h#L2175-L2177
(note: take care to look at the `F3DEX_GBI_2` one)

The disas script mostly assumes each command is one Gfx, so there's a single ptr++ at the end of the loop
But this command is two Gfx, so it skips two, and -1 to compensate for the "generic" ptr++
(note the second Gfx word is completely skipped regardless of what it is, it's just expected to be `gsMoveWd(G_MW_FORCEMTX,0,0x00010000)` I guess. Also note OoT doesn't use `gSPForceMatrix` and I'm not sure what purpose it has anyway)